app.js

```diff
@@ -74,6 +80,23 @@ app.use(cors());
 // adding morgan to log HTTP requests
 app.use(morgan("combined"));
 
+// swagger UI
+
+const options = {
+  definition: {
+    openapi: '3.0.0', // Specification (optional, defaults to swagger: '2.0')
+    info: {
+      title: 'G5API', // Title (required)
+      version: '0.1.0', // Version (required)
+    },
+  },
+  // Path to the API docs
+  apis: ['./routes/leaderboard.js','./routes/mapstats.js','./routes/users.js'],
+};
+const swaggerSpec = swaggerJSDoc(options);
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
```

Comment:
Maybe we should consider a different end point like `/docs/api` instead? I'm unsure, what are your thoughts?